packages/chakra-components/docs/03-Election.mdx

```diff
@@ -36,21 +36,33 @@ Show election title.
 
 ## Election Schedule
 
-Show election period. You can specify date format using [date-fns](https://date-fns.org/v2.30.0/docs/format) format.
+Show election period. You can specify date format using [date-fns](https://date-fns.org/v2.30.0/docs/format) format
 
 ```jsx live
 <ElectionSchedule format={'d-L-y HH:mm'} />
 ```
 
 ## Election Status Badge
 
-It shows election status badge. Possible values of the election are defined on `ElectionStatus` from `@vocdoni/sdk`:
+It shows election status badge. Possible values of the election are defined on `ElectionStatus` from [`@vocdoni/sdk`](https://github.com/vocdoni/vocdoni-sdk/blob/main/src/types/election/published.ts#L6-L18):
 `PROCESS_UNKNOWN`, `UPCOMING`, `ONGOING`, `ENDED`, `CANCELED`, `PAUSED`, `RESULTS`.
 
 ```jsx live
 <ElectionStatusBadge variant='solid' colorScheme='teal' />
 ```
 
+To style the [`ElectionStatusBadge`](https://chakra-ui.com/docs/components/tag) leverage the capabilities of the Chakra
+styled system. By simply adjusting the `colorScheme` property based on the election status will enable you to easily alter the
+color of the badge. This approach mirrors the one employed by the
+[default color scheme](https://github.com/vocdoni/ui-components/blob/main/packages/chakra-components/src/components/Election/StatusBadge.tsx#L12-L24)
```

Comment:
Same, let's use fixed sha commits for documentation links.

Reply:
Force pushed fix bfc1aa7